app/assets/javascripts/angular/directives/expression.js.erb

```diff
@@ -86,7 +86,7 @@ angular.module("Prometheus.directives").directive('expression', ["$http",
         }
 
         var expression = VariableInterpolator(scope.expr.expression, scope.vars);
-        var url = URLGenerator(server.url, "graph", scope.vars);
+        var url = URLGenerator(server.url, "/graph", scope.vars);
```

Comment:
Yes, totally unnecessary. It was added for consistency with the rest of the
project

Am Mittwoch, 23. September 2015 schrieb Tobias Schmidt :

> In app/assets/javascripts/angular/directives/expression.js.erb
> <https://github.com/prometheus/promdash/pull/456#discussion_r40226863>:
>
> > @@ -86,7 +86,7 @@ angular.module("Prometheus.directives").directive('expression', ["$http",
> >          }
> >
> >          var expression = VariableInterpolator(scope.expr.expression, scope.vars);
> > -        var url = URLGenerator(server.url, "graph", scope.vars);
> > +        var url = URLGenerator(server.url, "/graph", scope.vars);
>
> So, this change would not have been necessary actually?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/promdash/pull/456/files#r40226863>.
>